pkg/postage/interface.go

```diff
@@ -11,9 +11,16 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 )
 
+// SyncStatus interface definitions reflect the status of the postage syncing progress.
+type SyncStatus interface {
+	Get() (bool, error)
+	Set(error)
```

Comment:
`SetError` maybe but this is also a bit bizarre... an external component can set the state of an internal component via getter/setter?